src/pages/EventAdministration/components/EventParticipants.tsx

```diff
@@ -34,26 +45,29 @@ const Registrations = ({ onWait = false, eventId }: RegistrationsProps) => {
   );
   const showSnackbar = useSnackbar();
 
-  const getEmails = useCallback(() => {
-    let emails = '';
+  const getRegistrationDetails = ({ names, emails }: RegistrationsCopyDetails) => {
+    if (!names && !emails) {
+      return '';
+    }
     const participants = registrations;
-    participants.forEach((participant, i) => {
-      emails += participant.user_info.email;
-      if (i < participants.length - 1) {
-        emails += ' \n';
-      }
-    });
-    return emails;
-  }, [registrations]);
-
-  const copyEmails = () => {
+    return participants
```

Comment:
Kan bruke `registrations` direkte, usikker på hvorfor vi gjorde som vi gjorde før med å kopiere
```suggestion
    return registrations
```